libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -597,6 +597,27 @@ MAV_RESULT AP_Mount::handle_command_long(const mavlink_command_long_t &packet)
     }
 }
 
+/// Change the configuration of the mount
+void AP_Mount::handle_global_position_int(const mavlink_message_t *msg)
+{
+    mavlink_global_position_int_t packet;
+    mavlink_msg_global_position_int_decode(msg, &packet);
+
+    for (uint8_t instance=0; instance<AP_MOUNT_MAX_INSTANCES; instance++) {
+        if (_backends[instance] == nullptr) {
+            continue;
+        }
+        struct mount_state &_state = state[instance];
+        if (_state._target_sysid != msg->sysid) {
+            continue;
+        }
+        _state._target_sysid_location.lat = packet.lat;
```

Comment:
You may want to validate that the location passed in is actually a valid lat, lon. I think we have a helper for that already that AP_Mission uses....

Reply:
Sanity check added, thanks.